doc/docs/resources.md

```diff
@@ -2,9 +2,15 @@
 
 # Additional Resources
 
```

Comment:
Maybe a link to the workshop as well?

Reply:
Good point! How about a new section `Demos` and linking https://github.com/eclipse-bluechi/bluechi-demos?

Reply:
Updated. 
@mkemel PTAL

Reply:
Great